src/halmos/sevm.py

```diff
@@ -1747,12 +1757,23 @@ def create(
         loc: int = int_of(ex.st.pop(), "symbolic CREATE offset")
         size: int = int_of(ex.st.pop(), "symbolic CREATE size")
 
+        if op == EVM.CREATE2:
+            salt = ex.st.pop()
+
+        # lookup prank
+        caller = ex.prank.lookup(ex.this, con_addr(0))
+
         # contract creation code
         create_hexcode = wload(ex.st.memory, loc, size, prefer_concrete=True)
         create_code = Contract(create_hexcode)
 
         # new account address
-        new_addr = ex.new_address()
+        if op == EVM.CREATE:
+            new_addr = ex.new_address()
+        else:  # EVM.CREATE2
+            code_hash = ex.sha3_data(create_hexcode, create_hexcode.size() // 8)
```

Comment:
this fails when `create_hexcode` is concrete:

```
    code_hash = ex.sha3_data(create_hexcode, create_hexcode.size() // 8)
AttributeError: 'bytes' object has no attribute 'size'
```

```suggestion
            code_hash = ex.sha3_data(create_hexcode, byte_length(create_hexcode))
```